recipe/build.sh

```diff
@@ -29,6 +29,13 @@ if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
   extra_args="${extra_args} --enable-cross-compile --arch=$ARCH --target-os=$OS --cross-prefix=$HOST- --host-cc=$CC_FOR_BUILD"
 fi
 
+
+PKGCONF_LINKED=0
+if [[ ! -f ${BUILD_PREFIX}/bin/pkg-config ]]; then
+    ln -s ${BUILD_PREFIX}/bin/pkgconf ${BUILD_PREFIX}/bin/pkg-config
+    PKGCONF_LINKED=1
+fi
+
```

Comment:
@traversaro I think something like this is 'reasonable' while we know for a fact that it is worthwhile to create the conflict between packages.

Already we have revealed a usecase where `cairo.h` cannot be found??? Not sure if that is the actual error or an other problem.